client/me/purchases/remove-purchase/index.jsx

```diff
@@ -360,6 +388,7 @@ class RemovePurchase extends Component {
 					<Gridicon className="card__link-indicator" icon="trash" />
 				</Wrapper>
 				{ this.shouldShowNonPrimaryDomainWarning() && this.renderNonPrimaryDomainWarningDialog() }
+				{ this.shouldShowPlanWarning() && this.renderPlanWarningDialog() }
```

Comment:
I do not normally with this pattern of multiple render functions inside a component, but that seems to be the current approach of this file.
Personally I like to see a declarative component tree with lot's of ternaries and conditional logic which is much more readable at a glance. But YMMV.

Reply:
this was fixed